### PR TITLE
⭐️ add help description for installer

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

File: HELP.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+Install-Mondoo -Product cnspec;
+```
+
+# Package Information
+
+`https://install.mondoo.com/package/mondoo/{platform}/{arch}/{filetype}/{version}/{method}`
+
+The arguments support the following values
+  - **platform**: linux, windows, darwin
+  - **arch**:     amd64, arm64, armv7, armv6, 386
```
Comment (czunker) [inline HELP.md:71]: We also have `ppc64le`. Is this also supported in the installer?

File: HELP.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+### via Powershell (Windows)
+
+[`https://install.mondoo.com/ps1/cnspec`](https://install.mondoo.com/ps1/cnspec)
+
+```powershell
+Set-ExecutionPolicy Unrestricted -Scope Process -Force;
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072;
+iex ((New-Object System.Net.WebClient).DownloadString('https://install.mondoo.com/ps1/cnspec'));
+Install-Mondoo -Product cnspec;
+```
```
Comment (czunker) [inline HELP.md:63]: Shoud we add the login part from L51 also here?

**Review by czunker — COMMENTED** [review]
:tada: More docs!

File: HELP.md (hunk 1 above)
Comment (chris-rock) [reply HELP.md:71]: 100% great catch

File: HELP.md (hunk 2 above)
Comment (chris-rock) [reply HELP.md:63]: I think that is a great idea.

**Pull request merged**